### DISCUSSION TITLE
Is it possible to use `cockpit-machines` on ClearLinux?

### DISCUSSION THREAD

**Discussion by ThaDaVos** [cockpit-project/cockpit-machines#1708] [Q&A]
Is it possible to use `cockpit-machines` on ClearLinux? I can't find any bundle and having a hard time to figure out where Cockpit gets installed to, to manually install it

**Comment by martinpitt** [comment]
Nobody in our team has any experience with ClearLinux, so this is more of a community question. If it's not packaged, you can always install it from upstream of course (`sudo make install` or `make devel-install`), but you'll have to install all of its dependencies yourself (libvirt, libvirt-dbus, and others).
